src/test/java/com/puppycrawl/tools/checkstyle/utils/CheckUtilTest.java

```diff
@@ -443,6 +443,16 @@ public void testParseClassNames() {
             .isEqualTo(expected);
     }
 
+    @Test
+    public void testIsPackageInfo() {
+        assertWithMessage("Result is expected")
+            .that(CheckUtil.isPackageInfo("Bland.java"))
+            .isEqualTo(false);
```

Comment:
can we cover this by Inputs ?

Reply:
We can, but as the test shows, it doesn't matter. The contents of the file does not matter as this method doesn't read it.

Reply:
So we can make input file that has only empty class, so it will be clear that it doesn't matter.
But we do all by Inputs as much as possible.

Reply:
Done.

Reply:
Made non UT test.

Reply:
thanks a lot.